addons-l10n/en/mediarecorder.json

```diff
@@ -2,19 +2,20 @@
   "mediarecorder/record": "Record",
   "mediarecorder/stop": "Stop Recording",
   "mediarecorder/option-title": "Record Options",
-  "mediarecorder/record-duration": "Record Duration (in seconds)",
-  "mediarecorder/start-delay": "Start Delay (in seconds)",
+  "mediarecorder/record-duration": "End recording after:",
+  "mediarecorder/start-delay": "Start recording after:",
+  "mediarecorder/seconds": "seconds",
   "mediarecorder/starting-in": "Starting in {secs}...",
-  "mediarecorder/record-audio": "Include project sounds",
-  "mediarecorder/record-mic": "Include sounds from microphone",
+  "mediarecorder/record-audio": "Record project audio",
+  "mediarecorder/record-mic": "Record microphone audio",
   "mediarecorder/record-audio-description": "This does not include Text-to-Speech.",
-  "mediarecorder/record-after-flag": "Do not start recording until the Green Flag is clicked",
-  "mediarecorder/record-until-stop": "Stop recording after the project has stopped",
-  "mediarecorder/record-description": "Record the stage as a {extension} file. You can save it to your computer after the recording is finished.\nNote: variable and list monitors will not be visible.",
+  "mediarecorder/record-after-flag": "Wait until green flag is clicked",
+  "mediarecorder/record-until-stop": "End recording when project is stopped",
+  "mediarecorder/record-description": "Record a video of the stage and save it to your computer as a {extension} file after the recording is finished.\nNote: variable and list monitors will not be visible in the recording.",
```

Comment:
Maybe it's a good opportunity to 1. split this into two strings, and 2. make it easier to read?

Reply:
A possible alternative to the "monitors are not visible" note, is instead adding it as a checkbox that can't be enabled, and that clearly signals that the feature is not supported yet (for now... see other issues)

Reply:
I'll experiment with putting the note in a separate, highlighted sentence, kind of like this:

> Record a video of the stage and save it to your computer as a _{extension}_ file after the recording is finished.

> [!NOTE]
> Variable, list, and reporter monitors, as well as ask prompts will not be visible in the recording.

The note could also be put more concisely:

> [!NOTE]
> Some elements are not visible in recordings. <a href='#'>Learn more</a>

Which one do you prefer?

Reply:
> > [!NOTE]
> > Variable, list, and reporter monitors, as well as ask prompts will not be visible in the recording.

I like this version.

Reply:
Yes, the first one.